src/containers/Error/index.js

```diff
@@ -1,21 +1,25 @@
 import React, { PropTypes } from 'react';
 import IndexHeader from 'components/IndexHeader';
 import Helmet from 'react-helmet';
+import { FormattedHTMLMessage } from 'react-intl';
 
 const error = {
-  'invalid-surah': 'Surah is out of range',
-  'invalid-ayah-range': 'Ayah(s) selected are out of range'
+  'invalid-surah': "Surah is out of range. Please go to <a href='/'> home page</a> and select a Surah",
```

Comment:
@sabeurthabti @mmahalwy 
tried to convert this into JSX, got this.
Surah is out of range. Please go to [object Object] and select a Surah
```javascript
const homeLink = <Link to='/'> home page </Link>;
const error = {
  'invalid-surah': `Surah is out of range. Please go to ${homeLink} and select a Surah`,
  'invalid-ayah': `Ayah is out of range. Please go to ${homeLink} and select a Surah/Ayah`
};
```